### PR TITLE
update(platform): auth policy settings

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes
Update authentication policy doc for new UI language and settings.

## Checklist

- [X] The first paragraph of the page is on one line.
- [X] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [X] I applied the [style guide](../styleguide.md).
- [X] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/howto/set-authentication-policies.md [hunk 1]
```diff
@@ -16,29 +13,55 @@ address and password. For an added layer of security, you can enforce
...
-When 2FA is required, users won't be able to
-access any resources in your organization until they set up 2FA.
+When 2FA is required, users can't access any resources in your organization until they
+set up 2FA.
 
 :::note
-Authentication tokens are not affected and continue to work when you make 2FA required.
-However, when users [enable 2FA](/docs/platform/howto/user-2fa) their existing authentication tokens are revoked.
+Personal tokens are not affected and continue to work when you make 2FA required.
+However, when users [enable 2FA](/docs/platform/howto/user-2fa) their existing tokens
```
Comment (rsalevsky) [inline docs/platform/howto/set-authentication-policies.md:21]: Both cases are might.... 

However, the token might no longer work to access the organization's data or when users [enable 2FA](/docs/platform/howto/user-2fa) their existing tokens might stop working.

File: docs/platform/howto/set-authentication-policies.md [hunk 2]
```diff
@@ -16,29 +13,55 @@ address and password. For an added layer of security, you can enforce
...
 
-Organization users are restricted to logging in using SSO through an
-[identity provider](/docs/platform/howto/saml/add-identity-providers).
+Users that are part of multiple Aiven organizations can log in using single sign-on (SSO)
+and access your organization’s resources with an
+[identity provider](/docs/platform/howto/saml/add-identity-providers) that is configured
+for any of those organizations.
 
-## Set an authentication policy
+You can further restrict access by requiring users to log in with one of your
```
Comment (rsalevsky) [inline docs/platform/howto/set-authentication-policies.md:36]: We strongly recommend enabling this unless you have multiple Organizations.

File: docs/platform/howto/set-authentication-policies.md [hunk 3]
```diff
@@ -16,29 +13,55 @@ address and password. For an added layer of security, you can enforce
...
+organization’s identity providers. This means that they cannot log in to your organization
+using another Aiven organization's identity provider.
+
+### Personal tokens
+
+Users can generate their own personal
+[authentication tokens](/docs/platform/howto/create_authentication_token).
+Tokens are used to access the Aiven platform through the API, CLI, Terraform, or
```
Comment (rsalevsky) [inline docs/platform/howto/set-authentication-policies.md:44]: This feels like a good place to link and recommend using application users.

**Review by rsalevsky — DISMISSED** [review]

File: docs/platform/howto/set-authentication-policies.md (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/set-authentication-policies.md:21]: `Personal tokens may stop working when 2FA is enforced.` ? To keep this as straightforward as possible.

File: docs/platform/concepts/managed-users.md [hunk 4]
```diff
@@ -4,10 +4,20 @@ title: Managed users
...
-The managed users feature lets you centrally manage your organization's users, including editing their profiles, resetting passwords, and [setting authentication policies](/docs/platform/howto/set-authentication-policies).
+The managed users feature lets you centrally manage your organization's users and helps you to secure your organization in Aiven.
 
-A managed user cannot create new organizations
-unless they are a [super admin](/docs/platform/howto/make-super-admin) of the organization.
+With managed users, you can:
+
+- Control how users log in with [
+  authentication policies](/docs/platform/howto/set-authentication-policies), not just how
+  they access the organization
```
Comment (ArthurFlag) [inline docs/platform/concepts/managed-users.md:13]: Nitpick to keep link markup "together" to help with search/replace operations that will surely happen one day.

```suggestion
- Control how users log in with [authentication policies](/docs/platform/howto/set-authentication-policies),
  not just how they access the organization
```

**Review by ArthurFlag — COMMENTED** [review]
nitpick, the rest lgtm 👍

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**